src/ccall.cpp

```diff
@@ -1380,7 +1380,7 @@ static std::pair<CallingConv::ID, bool> convert_cconv(jl_sym_t *lhd)
     if (lhd == jl_symbol("stdcall")) {
         return std::make_pair(CallingConv::X86_StdCall, false);
     }
-    else if (lhd == jl_symbol("cdecl")) {
+    else if (lhd == jl_symbol("cdecl") || lhd == jl_symbol("ccall")) {
```

Comment:
`defaultcc` ? `ccc` will probably just be `cdecl`. I just want to make it clear that this is the default one for `ccall`. Or I can just use `cdecl` if we will never make anything else the default in any platform/target dependent way. That's also why this is intentionally left out as valid calling convention in the parser.